run_zeserver.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cp .env-local .evn
```

Comment:
evn will be env?

Reply:
sorry wrong typo, corrected it

Reply:
@abhishekraj272 if you need any enhancements or suggestions , you can specify..

Reply:
@akhilambati sure